src/Fixer/Casing/ConstantCaseFixer.php

```diff
@@ -110,12 +110,33 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             if (
                 $this->isNeighbourAccepted($tokens, $tokens->getPrevMeaningfulToken($index))
                 && $this->isNeighbourAccepted($tokens, $tokens->getNextMeaningfulToken($index))
+                && !$this->isEnumCaseName($tokens, $index)
             ) {
                 $tokens[$index] = new Token([$token->getId(), $fixFunction($token->getContent())]);
             }
         }
     }
 
+    private function isEnumCaseName(Tokens $tokens, int $index): bool {
```

Comment:
please run php-cs-fixer on itself. there are lots of issues in this file

Reply:
Too used to my own project's CS rules 😆